services/core-api/app/api/mines/variances/resources/variance.py

```diff
@@ -108,3 +109,18 @@ def put(self, mine_guid, variance_guid):
 
         variance.save()
         return variance
+
+    @api.doc(
+        description='Delete a variance.',
+        params={
+            'mine_guid': 'GUID of the mine to which the variance is associated',
+            'variance_guid': 'GUID of the variance to delete'
+        })
+    @requires_role_mine_admin
+    @api.response(204, 'Successfully deleted.')
+    def delete(self, mine_guid, variance_guid):
+        variance = Variance.find_by_mine_guid_and_variance_guid(mine_guid, variance_guid)
+        if variance is None:
+            raise NotFound('Unable to fetch variance')
```

Comment:
I think our common 404 message (like you did for permits above) would be better fitting.

Reply:
done